classes/cargo_bin.bbclass

```diff
@@ -106,22 +82,34 @@ cargo_bin_do_configure() {
     echo "#!/bin/sh" >"${WRAPPER_DIR}/linker-native-wrapper.sh"
     echo "${BUILD_CC} ${BUILD_LDFLAGS} \"\$@\"" >>"${WRAPPER_DIR}/linker-native-wrapper.sh"
     chmod +x "${WRAPPER_DIR}/linker-native-wrapper.sh"
-
-    # Create our global config in CARGO_HOME
-    create_cargo_config
 }
 
 cargo_bin_do_compile() {
     export TARGET_CC="${WRAPPER_DIR}/cc-wrapper.sh"
     export TARGET_CXX="${WRAPPER_DIR}/cxx-wrapper.sh"
     export CC="${WRAPPER_DIR}/cc-native-wrapper.sh"
     export CXX="${WRAPPER_DIR}/cxx-native-wrapper.sh"
-    export TARGET_LD="${WRAPPER_DIR}/ld-wrapper.sh"
-    export LD="${WRAPPER_DIR}/ld-native-wrapper.sh"
     export PKG_CONFIG_ALLOW_CROSS="1"
     export LDFLAGS=""
     export RUSTFLAGS="${RUSTFLAGS}"
-    export SSH_AUTH_SOCK="${SSH_AUTH_SOCK}"
```

Comment:
@sarfata Is the drop of this intentional?

Reply:
Nope - Good catch thanks. Just fixed.